tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -67,14 +70,46 @@ describe(`AI Lab extension installation and verification`, async () => {
     });
   });
   describe(`AI Lab extension verification`, async () => {
-    test(`Verify AI Lab is present in notification bar and open it`, async () => {
-      const aiLabNavBarItem: Locator = navigationBar.navigationLocator.getByLabel(AI_LAB_NAVBAR_EXTENSION_LABEL);
-      await playExpect(aiLabNavBarItem).toBeVisible();
-      await aiLabNavBarItem.click();
+    test(`Verify AI Lab is responsive`, async () => {
+      [page, webview] = await handleWebview();
+      const aiLabPage = new AILabPage(page, webview);
+      await aiLabPage.waitForPageLoad();
     });
-    test(`Verify AI Lab is running`, async () => {
-      const aiLabWebview: Locator = page.getByLabel(AI_LAB_PAGE_BODY_LABEL);
-      await playExpect(aiLabWebview).toBeVisible();
+    test(`Open Recipes Catalog`, async () => {
+      [page, webview] = await handleWebview();
```

Comment:
Is it really required to get the page and webview again here? Why can't the instances from the previous test be used? Are you navigating out of the webview somewhere that you need a new instance?

Reply:
fixed